lib/menubar/init.lua

```diff
@@ -121,7 +121,7 @@ menubar.left_label = "◀◀"
 local list_interspace = theme.xresources.apply_dpi(4) * 3
 
 --- Allows user to specify custom parameters for prompt.run function
--- (like colors).
+-- (like colors). These will override default parameters.
```

Comment:
but with your patch instead of overriding the default, it will _merge_ with defaults

Reply:
```suggestion
-- (like colors). This will merge with the default parameters, overriding affected values.
```